DashSync/shared/Models/DAPI/Networking/DSDAPIGRPCResponseHandler.m

```diff
@@ -312,8 +312,14 @@ - (void)parseGetIdentitiesByPublicKeyHashesMessage:(GetIdentitiesByPublicKeyHash
 
         for (NSData *cborData in getIdentitiesResponse.identitiesArray) {
             if (!cborData.length) continue;
-            NSData *identityData = [cborData subdataWithRange:NSMakeRange(4, cborData.length - 4)];
+            NSArray<NSData *> *arrayOfIdentities = [cborData ds_decodeCborError:&error];
+            if (arrayOfIdentities.count == 0) continue;
+            NSData *identityData = arrayOfIdentities.firstObject;
+            uint32_t version = [identityData UInt32AtOffset:0];
+            identityData = [arrayOfIdentities.firstObject subdataWithRange:NSMakeRange(4, identityData.length - 4)];
```

Comment:
This looks quite bad, but I think it's more an issue in V22 than here.

Reply:
I agree, but that's how the data is encoded now

Reply:
Made it look a bit nicer